content/blog/entries/integartion-vocabulary-ccos/contents.lr

```diff
@@ -0,0 +1,33 @@
+title: Integration of Vocabulary with CCOS.
+---
+categories:
+outreachy
+tech
+open-source
+---
+author: dhruvi16
+---
+pub_date: 2020-07-08
+---
+body:
+
+
+This blog demonstrates the project I am working on during the course of my Outreachy internship. My project involves redesigning and reimplementing one of the web products of the CC network using our new cohesive design library — [Vocabulary](https://cc-vocabulary.netlify.app/?path=/docs/vocabulary-introduction--page). After completion of my internship, the [Creative Commons Open Source](https://opensource.creativecommons.org/) website will have a completely different looking interface that will extensively use Vocabulary components.
+
+## The Problem -
+
+There exists a large variety of web products across the Creative Commons network. These products serve different purposes for the network and so do they vary in the way of presenting the existing content. So, to conquer the inconsistency across the different products we have our coherent design system called Vocabulary.
+
+If we come to the current CC Open Source website, it is quite off-track considering our new design system Vocabulary. The styles and components of the website are not well aligned with the components of the Vocabulary. And thus it lacks harmony and consistency concerning the brand.
+
+The purpose of a design system gets dissolved if it is not well incorporated with the products and we do have this problem with the CC Open Source website. Hence, there was a need for a redesign of CCOS as per the new Design Library. This project aims to rectify the problems and inconsistencies of the CCOS website and build a completely new website (as per Vocabulary).
+
+## The Solution -
+
+I started with trying out different mock-ups for the new website in Figma. You can see what I tried to make [here](https://www.figma.com/file/ka3zs1iYnqJvyLnvAV3cW7/Home-proposals). And then with the help of our UX designer Francisco, we created all the mock-ups and finalized how the website will look in a couple of weeks, here the [link](https://www.figma.com/file/mttcugI1UxvCJRKE5sdpbO/Mockups?node-id=759%3A516) for that.
+
+I started my internship with going through the website’s tech stack, the website is made using Lektor, with which I was not very familiar. I read the official docs went through projects, tried making demo projects, and took help from mentors wherever I got stuck.
+
+To keep the project clean I added a theme that only has templates for the project and alters no content (as per aim). While updating templates, some components were not available in the Vocabulary yet, so I worked on adding those components to the library. And then used them in my new templates. You can see the ongoing work [here](https://cc-open-source-dev.netlify.app/).
```

Comment:
I think to make that Url public could lead to confusion, so I would say let's keep it just for us. Do you have any thoughts on that?